src/main/java/com/thealgorithms/datastructures/trees/TreeTraversal.java

```diff
@@ -104,16 +106,16 @@ public void printPostOrder() {
      */
     public void printLevelOrder() {
         LinkedList<Node> queue = new LinkedList<>();
-        queue.add(this);
+        queue.add(null); // What node should be added here ? This had a wrong parameter type
```

Comment:
NodeTree

Reply:
there is a left and a right node and if you want to use the NodeTree using this u still have to tell what part of the object to use as given parameter. The method add() asks for type Node e and not an Object. thats how far i understand it.
Either you may change the methods input parameter to Object or use a Node e. NodeTree would be an object.
I can help where i can but im still learning the stuff at my university :)

Reply:
You can make the queue of type LinkedList<TreeNode>